.github/workflows/website.yaml

```
@@ -0,0 +1,31 @@
+name: Website deploy
+on:
+  push:
+    branches:
+    - stable
+
+jobs:
+  website-nightly:
```

```suggestion
  update-website:
```
I guess this CI is not the `nightly` type.